src/engine.cpp

```diff
@@ -4,11 +4,13 @@
  */
 
 #include "engine.h"
+#include "log.h"
 
 
 volatile bool Engine::_enabled = false;
 
 Error Engine::check(Arguments& args) {
```

Comment:
`Engine::check` and all of its overridden versions were only called from `Profiler::check`.
Since `Profiler::check` is now stubbed, you don't need `Engine::check` anymore.